src/trading/getSuggestedSlippage.ts

```diff
@@ -0,0 +1,31 @@
+import { getSigner } from 'src/common/utils/wallet'
+import { OrderBookApi, OrderQuoteResponse } from '../order-book'
+import { DEFAULT_SLIPPAGE_BPS } from './consts'
+import { getQuoteRaw, getTrader } from './getQuote'
+import { QuoterParameters, SwapAdvancedSettings, SwapParameters, TradeParameters } from './types'
+
+export interface SuggestSlippageBps {
+  tradeParameters: TradeParameters
+  quote: OrderQuoteResponse
+  trader: QuoterParameters
+  advancedSettings?: SwapAdvancedSettings
+}
+
+export function suggestSlippageBps(_params: SuggestSlippageBps): number {
+  // TODO: Do something really smart here :)
+  return DEFAULT_SLIPPAGE_BPS
+}
+
+export async function getSuggestedSlippageBpsWithSigner(
```

Comment:
I'm not sure about a separate function for that.
Maybe we can include smart slippage in existing `QuoteResults`?

Reply:
I thought it was good to give a standalone method (just to allow them to have their own slippage logic, but load independently the fee), but I will delete probably good enough without it